### PR TITLE
tweaks to consumption code

### PR DESCRIPTION
**Pull request by snopoke**
@sravfeyn 
cc @czue

### CODE REVIEW

**Comment by czue** [comment]
build fails look real
```
======================================================================
ERROR: test_sql_objects_deletion (corehq.apps.domain.tests.test_delete_domain.TestDeleteDomain)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/domain/tests/test_delete_domain.py", line 63, in setUp
    self._create_data('test', i)
  File "/mnt/corehq/apps/domain/tests/test_delete_domain.py", line 36, in _create_data
    stock_on_hand=100
  File "/vendor/lib/python2.7/site-packages/django/db/models/manager.py", line 92, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 372, in create
    obj.save(force_insert=True, using=self.db)
  File "corehq/ex-submodules/casexml/apps/stock/models.py", line 86, in save
    force_insert=force_insert, force_update=force_update, using=using, update_fields=update_fields
  File "/vendor/lib/python2.7/site-packages/django/db/models/base.py", line 589, in save
    force_update=force_update, update_fields=update_fields)
  File "/vendor/lib/python2.7/site-packages/django/db/models/base.py", line 626, in save_base
    update_fields=update_fields, raw=raw, using=using)
  File "/vendor/lib/python2.7/site-packages/django/dispatch/dispatcher.py", line 198, in send
    response = receiver(signal=self, sender=sender, **named)
  File "corehq/ex-submodules/casexml/apps/stock/signals.py", line 19, in stock_transaction_post_save_signal_receiver
    update_stock_state_for_transaction(instance)
  File "corehq/ex-submodules/casexml/apps/stock/signals.py", line 126, in update_stock_state_for_transaction
    consumption_calc
  File "corehq/ex-submodules/casexml/apps/stock/consumption.py", line 94, in compute_daily_consumption
    return compute_daily_consumption_from_transactions(transactions, window_start, configuration)
  File "corehq/ex-submodules/casexml/apps/stock/consumption.py", line 216, in compute_daily_consumption_from_transactions
    periods = list(split_periods(transactions))
  File "corehq/ex-submodules/casexml/apps/stock/consumption.py", line 197, in split_periods
    if tx.is_checkpoint:
  File "corehq/ex-submodules/casexml/apps/stock/models.py", line 51, in is_checkpoint
    return self.type == const.TRANSACTION_TYPE_STOCKONHAND and not self.is_stockout
  File "corehq/ex-submodules/casexml/apps/stock/models.py", line 45, in is_stockout
    (self.type == const.TRANSACTION_TYPE_STOCKONHAND and self.consumption_value == 0) or
AttributeError: 'StockTransaction' object has no attribute 'consumption_value'
----------------------------------------------------------------------
```

File: corehq/ex-submodules/casexml/apps/stock/tests/mock_consumption.py [hunk 1]
```diff
@@ -5,7 +6,10 @@
 
 to_ts = dateparse.json_format_datetime
 
-MockTransaction = collections.namedtuple('MockTransaction', ['action', 'value', 'received_on'])
+class MockTransaction(
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/stock/tests/mock_consumption.py]: E302 expected 2 blank lines, found 1

File: corehq/ex-submodules/casexml/apps/stock/tests/mock_consumption.py [hunk 2]
```diff
@@ -5,7 +6,10 @@
...
+    collections.namedtuple('MockTransaction', ['type', 'normalized_value', 'received_on']),
+    ConsumptionMixin):
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/stock/tests/mock_consumption.py]: E125 continuation line with same indent as next logical line

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by czue** [comment]
nice cleanup!

**Pull request merged**